### ISSUE TITLE
nothing is clickable when app is opened on Xiaomi HyperOS [1.0.3.0.TGDMIXM] using bitwarden android version [2024.4.0]

### ISSUE THREAD

**Issue by kashankrm** [bitwarden/mobile#3180]
### Steps To Reproduce

1. open app [first time opening it after install]
2. click on text field to write your email

### Expected Result

it should let me type my email and select the server.

### Actual Result

it does not let me type my email, when clicked it does not open the keyboard to type. I also clicked the change server option and that is also not working, it seems nothing is clickable. 

### Additional Context

I have tried multiple versions and none work, all have the same problem.

I am guessing this is somehow related to my OS. I would like some guidance how should I go about debugging this?

### Operating System

Android

### Device

redmi note 11 pro

### Build Version

2024.4.0

### Beta

- [ ] Using a pre-release version of the application.

**Comment by SergeantConfused** [comment]
Hi @kashankrm,

Thank you for this report. This reminds me of [this one](https://github.com/bitwarden/mobile/issues/1899).
1. Could you please rotate the phone into landscape mode and let me know if that helps?
2. Do you have any accessibility services running?

Thank you in advance,

**Comment by kashankrm** [comment]
Hey it is strange, yesterday I tried your suggestion and they didn't work. I checked and there were no accessibility services running as well.

but today it works fine (both old and new version). 

One thing I did yesterday was I turned on the accessibility service for downloaded apps for bitwarden (it said it helps in entering password on different apps). 

I may have restarted my phone since yesterday. But today the app works fine. 

Its really strange because the app used to work fine before then randomly I had this issue and now it works again. I really don't know if the steps I wrote above were the reason or there was some other reason for it working.

Thanks for the quick reply, I will close this issue and if I find this problem again I will open it again and then we can debug it again?
